data/human/hails.txt

```diff
@@ -1883,6 +1883,23 @@ phrase "friendly civilian"
 		", a pretty insignificant company located on just one planet."
 		", but next time, I'll buy from the Syndicate again."
 
+phrase "friendly civilian"
+	word
+		"I heard that "
+		"I read a rumor that "
+		"I found out that "
+	word
+		"if you have "
+		"you need to have "
```

Comment:
This option ends up being a run on sentence. Fix would be to add conjunctions at the start of the last phrase, but that would mess up the other option at this level. So unless there's a way to arrange that with substitutions, I'm thinking these two just aren't compatible here.

Reply:
The option has been removed.